client/src/components/forms/CommentEditor.vue

```diff
@@ -11,7 +11,7 @@
     <div class="comment-editor">
       <editor-content :editor="editor" />
     </div>
-    <div v-if="props.isInlineComment" class="q-py-md">
+    <div v-if="props.commentType === 'inline'" class="q-py-md">
```

Comment:
```suggestion
    <div v-if="commentType === 'inline'" class="q-py-md">
```

> Declared props are automatically exposed to the template. defineProps also returns an object that contains all the props passed to the component, so that we can access them in JavaScript if needed.
https://vuejs.org/guide/essentials/component-basics.html#passing-props